src/renate/updaters/model_updater.py

```diff
@@ -382,6 +385,7 @@ def _load_learner(
             self._learner_state_file,
             model=self._model,
             logged_metrics=self._logged_metrics,
+            strict=False,
```

Comment:
Why?

Reply:
Let me check if it is necessary. But we still try to load the checkpoint and since keys are missing in `state_dict`, it complains otherwise.